planner/src/main/java/com/dask/sql/application/RelationalAlgebraGenerator.java

```diff
@@ -1,10 +1,13 @@
 package com.dask.sql.application;
 
 import java.sql.SQLException;
+import java.util.ArrayList;
 import java.util.List;
 
 import com.dask.sql.schema.DaskSchema;
 
+import org.apache.calcite.plan.RelOptRule;
+import org.apache.calcite.plan.RelOptRuleCall;
```

Comment:
Maybe I'm missing something, but could we remove this import? Looks like it's not being used anywhere in this file

Reply:
`RelOptRuleCall` can be removed but `RelOptRule` is used in the constructor. I'll make that change now